docs/install/plugins.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Standalone Plugin Install
+parent: Install and Configure
+nav_order: 98
+---
+
+# Standalone plugin install
+
+If you don't want to use the all-in-one Open Distro for Elasticsearch Docker images or RPM packages, you can install the Security, Alerting, SQL, and Performance Analyzer plugins on a compatible Elasticsearch cluster just like any other Elasticsearch plugin. Navigate to the Elasticsearch home directory (likely `/usr/share/elasticsearch`) and run:
+
+```bash
+sudo bin/elasticsearch-plugin install https://d3g5vo6xdbdb9a.cloudfront.net/downloads/elasticsearch-plugins/opendistro-security/opendistro_security-0.8.0.0.zip
+sudo bin/elasticsearch-plugin install https://d3g5vo6xdbdb9a.cloudfront.net/downloads/elasticsearch-plugins/opendistro-alerting/opendistro_alerting-0.8.0.0.zip
+sudo bin/elasticsearch-plugin install https://d3g5vo6xdbdb9a.cloudfront.net/downloads/elasticsearch-plugins/opendistro-sql/opendistro_sql-0.8.0.0.zip
+sudo bin/elasticsearch-plugin install https://d3g5vo6xdbdb9a.cloudfront.net/downloads/elasticsearch-plugins/performance-analyzer/opendistro_performance_analyzer-0.8.0.0.zip
+```
+
+See [Version history](../../../#version-history) for the versions of Elasticsearch that Open Distro for Elasticsearch supports. You must have the exact compatible version installed (e.g. 6.6.2 and not 6.6.1). To get a list of available Elasticsearch versions on CentOS 7 and Amazon Linux 2:
+
+```bash
+sudo yum --showduplicates list elasticsearch
+```
+
+Then you can specify the version you need:
+
+```bash
+sudo yum install elasticsearch-6.6.2-1
```

Comment:
In order to this, we need to specify the RPM repo location. Should we mention it here?

https://www.elastic.co/guide/en/elasticsearch/reference/6.3/rpm.html#rpm-repo

Also, I think the very ending `-1` is not necessary.

Reply:
I'd rather operate under the assumption that people have an existing cluster and not get too far into the weeds on installing ES on your own.

I'll definitely remove the unnecessary -1, though. It works, too, but is ugly -- slipped in during my copy and pasting.